recipes/featuretools_sql/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "featuretools_sql" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/featuretools_sql-{{ version }}.tar.gz
+  sha256: 1b3d670921e5371ae031320f55e0408689bcf8658b8d0b73714512e76520526c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - pip
+  run:
+    - python >=3.8,<4.0
+    - numpy >=1.17.5
+    - pandas >=1.3.0
+    - featuretools >=1.5.0
+    - psycopg2 >=2.9.3
+    - sqlalchemy >=1.4.2
+    - PyMySQL >=1.0.2
+    - snowflake-sqlalchemy >=1.4.2
+
+test:
+  imports:
+    - featuretools_sql
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  doc_url: https://featuretools.alteryx.com/en/stable/guides/sql_database_integration.html
+  dev_url: https://github.com/alteryx/featuretools-sql
+  home: https://pypi.org/project/featuretools-sql/
+  summary: directly import your relational data into Featuretools to run automated feature engineering
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - stefaniesmith
```

Comment:
The users here need to acknowledge that they like to be maintainers.

Reply:
I would like to be a maintainer.

Reply:
I would like to be a maintainer.

Reply:
I would like to be a maintainer.

Reply:
I would like to be a maintainer.

Reply:
I would like to be a maintainer.

Reply:
I would like to be a maintainer.

Reply:
I would like to be a maintainer.

Reply:
I would like to be a maintainer.

Reply:
I would like to be a maintainer.

Reply:
@xhochy All maintainers have commented.